### ISSUE TITLE
Remote cluster feature

### ISSUE THREAD

**Issue by tschale** [Blueshoe/pytest-kubernetes#3]
It would be nice to have the possibility to run tests on a remote cluster, instead of a local (minikube, k3s, ...) one. 

Advantages:
- less setup time, as the remote cluster already exists (i.e. overall shorter testrun time)
- no need for specific setup/workaround for local provider (i.e. higher testing-staging-prod parity)

**Comment by egegunes** [comment]
Yes, I'd love to see this feature too since we need to run our tests on different platforms (GKE, EKS, Azure, Openshift, minikube etc.). To be honest I don't think this should be handled by the tool and make user pass a working kube config.
